app/directives/accessControlForm/porAccessControlFormController.js

```diff
@@ -58,10 +58,13 @@ function ($q, UserService, Notifications, Authentication, ResourceControlService
     })
     .then(function success(data) {
       if (data) {
+        ctrl.formData.AccessControlEnabled = true;
```

Comment:
The data here is undefined for a new container, so default to access control enable
But the data is undefined too with a from container in public mode, and the form initialize with default access control enable (instead of disable)